importer/NlAwNl.py

```diff
@@ -0,0 +1,93 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class NlAwNl(Monument):
+
+    def set_address(self):
+        if self.has_non_empty_attribute("adres"):
+            if utils.contains_digit(self.adres):
+                town = utils.remove_markup(self.plaats)
+                address = "{}, {}".format(self.adres, town)
+                self.add_statement("located_street", address)
+            else:
+                self.add_to_report("adres", self.adres, "located_street")
+
+    def update_labels(self):
+        nl = utils.remove_markup(self.omschrijving)
+        self.add_label("nl", nl)
+
+    def update_descriptions(self):
+        desc = "cultural heritage monument in Aruba"
+        self.add_description("en", desc)
+
+    def set_adm_location(self):
+        aruba = "Q21203"
+        self.add_statement("located_adm", aruba)
+
+    def set_location(self):
+        loc_q = None
+        loc_dic = self.data_files["settlements"]
+        if self.has_non_empty_attribute("plaats"):
+            if utils.count_wikilinks(self.plaats) == 1:
+                loc_q = utils.q_from_first_wikilink("nl", self.plaats)
+            else:
+                loc_match = [x for x in loc_dic if x[
+                    "itemLabel"] == self.plaats]
+                if len(loc_match) == 1:
+                    loc_q = loc_match[0]["item"]
+        if loc_q:
+            self.add_statement("location", loc_q)
+        else:
+            self.add_to_report("plaats", self.plaats, "location")
+
+    def set_inception(self):
+        if self.has_non_empty_attribute("bouwjaar"):
+            if utils.legit_year(self.bouwjaar):
+                inc_year = {"time_value": {"year": self.bouwjaar}}
+                self.add_statement("inception", inc_year)
+            else:
+                self.add_to_report("bouwjaar", self.bouwjaar, "inception")
+
+    def set_heritage_id(self):
```

Comment:
I'm unsure if we might actually be able to use `AW-` as a prefix here since it is the official iso for the country (within the other country)

Reply:
So that's either NL for the real country, AW for the "country" or NL-AW for the database prefix. Personally I think it either the first or the last one because they're easy to explain, as opposed to "item with a ISO 3166-1 alpha-2 code which may or may not be a sovereign country".

Reply:
After looking at the nl-gem case I think the lat inte id's best since it is within the wlm dataset that uniqueness is guaranteed.

Reply:
Changed to WLM prefix. Will also affect other datasets where table name != country iso.